File: codekata/basics/triangle.py
#to check the numbers form a triangle 
#function to find the triangle
def tri(a,b,c):
 #condition to check the sides form triangle
    if(a+b>=c) or (a+c>=b) or (b+c>=a):
    	return "yes"
    else:
      	return "no"
#get the sides of the triangle from the user
x,y,z=map(int,input().split())
print(tri(x,y,z))

File: java_script/codekata/basics/triangle.js
//to sheck whether the 3 sides form a triangle 
const readline = require('readline');

const inp = readline.createInterface({
  input: process.stdin
});

const a = [];

inp.on("line", (data) => {
	a.push(data);
});

inp.on("close", () => {
    var x=a[0].split(" ");
    var b=parseInt(x[0]);
    var c=parseInt(x[1]);
    var d=parseInt(x[2]);
    if((b+c)<=d || (c+d)<=b || (b+d)<=c)
    {
        console.log("no");
    }
    else
    {
        console.log("yes");
    }
});
